recipes/tinuous/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "tinuous" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 597a43c00c2512b5a82034adc31c36ca84633010f6c186e153b3f438b4ce1271
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Add:
```yaml
  entry_points:
    - tinuous = tinuous.__main__:main
```
See [here](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) for more information.